src/event-progress-bar.js

```diff
@@ -0,0 +1,63 @@
+// Update the count down every 1 second
+var x = setInterval( doProgress, 1000 );
+
+function doProgress() {
+
```

Comment:
Why `var` in some places and `let` in others?

Reply:
Simply because I didn't know the difference.
Since then we have re-worked the JavaScript code with Stephen, so this is done.
Done.